articles/managed-services/man-patching-faq.md

```diff
@@ -0,0 +1,152 @@
+
+## Support
+
+### How do I raise a support ticket?
+
+The secure online [UKCloud Portal](https://portal.ukcloud.com/login) provides most common service management functionality. You can raise tickets within the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the Portal. Alternatively, you can contact support by phone or email.
+
+### What are your service maintenance windows?
+
+As far as possible, planned maintenance of UKCloud's infrastructure takes place periodically and at a time that aims to minimise customer impact. We provide customers with at least 14 days' advance notice of planned maintenance.
+
+As far as possible, emergency maintenance of UKCloud's infrastructure takes place between the hours of 00:00 and 06:00 (UK local time) Monday to Friday, or between the hours of 08:00 and 12:00 (UK local time) on Saturday or Sunday, unless there is an identified and demonstrable immediate risk to a customer's environment. Whenever possible, we provide customers with at least six hours' advance notice of emergency maintenance.
+
+### Can I configure email alerts from the Portal?
+
+Yes, you can have Portal notifications sent to you at the email address associated with your Portal login. Notifications provide information about updates to UKCloud services, in addition to maintenance and incident notifications.
+
+## Onboarding
+
+### How can I get started with the service?
+
+Within 5 business days of accepting an order, UKCloud will create the customer's patching environment and commence patching any environments in-line with the Device Group schedules they have opted-in to.
+
+### Is there a free trial?
+
+Due to the initial complexity of configuring the patching environment, UKCloud is currently unable to offer a free trial of this service.
+
+## Billing and legal
+
+### What is the smallest unit of time I will be billed for?
+
+The minimum unit of time for use is monthly. Part months will be rounded up.
+
+### Will I be charged for resources in a 'Shut Off' state?
+
+UKCloud will continue to charge for any resources that you have elected to be monitored regardless of the state they're in. In order to stop any charges, you'll need to raise a ticket within the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the Portal. Alternatively, you can contact support by phone or email.
+
+### How can I view billing information?
+
+Online visibility of real-time or historic billing information is currently not available as a service provided by UKCloud.
+
+### How can I pay for the services?
+
+Payment for UKCloud services can be made by direct bank transfer (BACS/CHAPS). You can find bank details for payment on your invoice.
+
+If you signed up with a credit or debit card, your payment card will be automatically charged no sooner than seven (7) days after the invoice date.
```

Comment:
Can customer use credit card onboarding for this service?

Reply:
Ooooh, that's a very good point! I would say with the way CCO is going (we may pull it!), probably best that I remove this line.
I'll also go back and check the MMaaS FAQ as I think it's in there.